onward/app/views/fragments/richLinkBody.scala.html

```diff
@@ -78,41 +65,19 @@ <h1 class="rich-link__title">
             </div>
         }
 
-        @if(!experiments.ActiveExperiments.isParticipating(experiments.Garnett)) {
-            @if(!content.tags.isGallery && !content.tags.isVideo) {
-                <div class="rich-link__standfirst u-cf">
```

Comment:
`rich-link__standfirst` and later `rich-link__byline-img` and `rich-link__byline-img__img`: these seem to be the only occurrences of these classes in the templates... perhaps we can also delete the corresponding CSS rules from the stylesheets?

Reply:
If you're happy to do this... 👍